spatialos-sdk/src/worker/mod.rs

```diff
@@ -32,9 +33,11 @@ impl EntityId {
     pub fn is_valid(self) -> bool {
         self.id > 0
     }
+}
 
-    pub fn to_string(self) -> String {
-        format!("EntityId: {}", self.id)
+impl Display for EntityId {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), Error> {
+        f.write_fmt(format_args!("EntityId: {}", self.id))
```

Comment:
You can use the `write!` macro here to make it a bit cleaner, I think:

```suggestion
        write!(f, "EntityId: {}", self.id)
```

Reply:
Nice! :)